templates/install.html

```diff
@@ -4,102 +4,72 @@
             </div>
         </div>
     </section>
-    <section class="row clearfix instruction">
-        <div class="inner-wrapper">
-            <div class="five-col instruction__bullet">
-                <div class="one-col">
-                    <span class="instruction__step">1</span>
-                </div>
-                <div class="four-col last-col">
-                    <h3 class="instruction__title">Setup your hardware</h3>
-                </div>
-            </div>
-            <div class="seven-col last-col instruction__details">
-                <p>You need one small server for MAAS and at least one server which can be managed with a <abbr title="Baseboard management controller">BMC</abbr>. It is recommended to have the MAAS server provide <abbr title="Dynamic Host Configuration Protocol">DHCP</abbr> and <abbr title="Domain name service">DNS</abbr> on a network the managed machines are connected to.</p>
-            </div>
-        </div>
-    </section>
-    <section class="row clearfix instruction">
-        <div class="inner-wrapper">
-            <div class="five-col instruction__bullet">
-                <div class="one-col">
-                    <span class="instruction__step">2</span>
-                </div>
-                <div class="four-col last-col">
-                    <h3 class="instruction__title">Install Ubuntu Server</h3>
-                </div>
-            </div>
-            <div class="seven-col last-col instruction__details">
-                <p><a href="http://www.ubuntu.com/download/server" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download Ubuntu Server', 'eventAction' : 'Click', 'eventLabel' : 'Download Ubuntu Server 16.04 LTS' });" class="external">Download Ubuntu Server 16.04 LTS</a> and follow the step-by-step installation instructions on your MAAS server.</p>
-            </div>
-        </div>
-    </section>
-    <section class="row clearfix instruction">
-        <div class="inner-wrapper">
-            <div class="five-col instruction__bullet">
-                <div class="one-col">
-                    <span class="instruction__step">3</span>
-                </div>
-                <div class="four-col last-col">
-                    <h3 class="instruction__title">Install MAAS</h3>
-                </div>
-            </div>
-            <div class="seven-col last-col instruction__details">
-                <p class="command-line">
-                    <input class="command-line__input" value="sudo apt update" readonly="readonly">
-                    <button class="js-copy-to-clipboard command-line__copy-button">Copy to clipboard</button>
+    <section class="p-strip is-bordered is-deep">
```

Comment:
This would be better with `p-strip--light` as the details sections are on white and it helps see a separation of sections.